app/db/models/lawpolicy/family.py

```diff
@@ -71,8 +71,8 @@ class DocumentType(Base):
     """
 
     __tablename__ = "document_type"
-
-    document_type = sa.Column(sa.Text,  primary_key=True)
+    id = sa.Column(sa.Integer, primary_key=True)
```

Comment:
So why are we using an id column of int here where we have the unique name column - is it incase we change the name of the doc type?

Reply:
So this has been hoyked out of the deprecated model - and as such I didn't really want to change it - but you make a good point /cc @joel-wright ?

Reply:
Agreed, the name should be a unique type in this table